lib/css/components/_stacks-tables.less

```diff
@@ -315,6 +314,23 @@
             opacity: 0.3;
         }
     }
+
+    //  $$  Responsive
+    //  ------------------------------------------------------------------------
+    &.s-table__responsive {
+        display: block;
+        overflow-x: auto;
+        @scrollbar-styles(); // Style the scrollbars appropriately
+    }
+
+    &.s-table__sticky {
+        th:first-child {
```

Comment:
I've found a potential show stopper with this approach. We lose a lot of borders in Firefox.

![image](https://user-images.githubusercontent.com/1369864/84062837-9a23bf80-a985-11ea-9dbf-df661b13df90.png)

Reply:
Firefox mobile? Or just desktop shrunk down?

Reply:
Ah nevermind I was looking in the wrong branch. I recreated.

There is actually another issue too - we need to set the bg colour (which I'm doing) but it's not dark mode respecting right now

Reply:
I switched the background color to `var(--white)` which makes that column appear darker in dark mode, since the rest of the table is actually transparent. This means we can't reliably choose something like `background-color: inherit`. We could be more explicit on the entire table (or each `td` and `th`) by setting their background colors `var(--white)`, but we'd still have the issue with the borders disappearing.

I wonder if instead of using borders, we actually just use box shadows 🤔

Reply:
Yeah I can investigate. 

This too: https://stackoverflow.com/questions/266877/disappearing-css-table-cell-borders-in-gecko-based-browsers